vispy/visuals/circle.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+import numpy as np
+import math
+
+from .mesh import MeshVisual
+
+
+class CircleVisual(MeshVisual):
+    """Visual that calculates and displays a circle using triangle's
+
+    Parameters
+    ----------
+    corners : int
```

Comment:
I'm not sure I understand, why "corners"? Are these the "wedges" of the pie chart? What units are these in? Degrees?

Oh wait, it says histogram, but do you ever run the data through a histogram? No wait, this is an integer?

Oh, looking at the example so `color` is really what determines the size of the wedges? And `corners` is more of a "resolution"? No...wait...I'm very confused.

Reply:
Oh yes corners are a bit confosing I called it corners because its not actually a circle its a "n-gon", like a octogon or a pentagon. For example an octogon would have 8 corners, and a pentagon would have 5. For the pie chart I set it to 100, so its more like a 100-gon. So yes its made up of wedges, and each wedge can be individualy colored.

Regarding the histogram thats my bad I made this while having covid and lying in bed and I copyed the histogram.py Parameters comment and forgot to change the line with the histogram - im sorry for the confusion